spring-cloud-starter-dataflow-server-local/src/test/java/org/springframework/cloud/dataflow/server/local/security/LocalServerSecurityRootWithUsersFileTests.java

```diff
@@ -49,6 +50,7 @@
 	private static UserCredentials fullUser = new UserCredentials("fulluser", "fullpassword");
 
 	@Test
+	@Ignore("Revisit once Outh2/OpenID Connect security is implemented")
```

Comment:
Was this waiting on https://github.com/spring-cloud/spring-cloud-common-security-config/commit/cb190c3c80b4904090b13224e419e17d8e1238cb ?